src/Surfnet/Stepup/Identity/Api/Identity.php

```diff
@@ -224,22 +224,24 @@ public function accreditWith(
     );
 
     /**
+     * @param Institution $institution
      * @param RegistrationAuthorityRole $role
      * @return void
      */
-    public function appointAs(RegistrationAuthorityRole $role);
+    public function appointAs(Institution $institution, RegistrationAuthorityRole $role);
```

Comment:
You could break from the norm and add some additional context to the added parameters in the docblocks. For now I miss insights into why the institution is a requirement for these methods.

Reply:
Added a doc block

Reply:
Added a doc block

Reply:
There is an error in that doc block:

>  This method will appoint an institution to become ra or raa for another institution

The institution is not appointed right? The Identity is.

Reply:
That addition does not make that much sense:

> This method will appoint an institution to become ra or raa for another institution

The identity is appointed, not the institution.